glusterd2/xlator/options/options.go

```diff
@@ -160,7 +160,8 @@ func ValidateBool(o *Option, val string) error {
 
 // ValidateClientAuthAddr validates mount auth address
 func ValidateClientAuthAddr(o *Option, val string) error {
-	return ErrInvalidArg
+	err := ValidateInternetAddress(o, val)
```

Comment:
I missed this earlier. `ClientAuthAddr` is supposed to be a comma-seperated list. So you'll need to iterate through the list. Since this is the same as what you do for `ValidInternetAddressList`, just call `ValidateInternetAddressList` for `case OptionTypeClientAuthAddr` in `(*Option)Validate()`.

Reply:
@kshlm rectified the same as suggested. Please review the same. Thank you.